contracts/sol6/mock/dao/MockChainLinkSanityRate.sol

```diff
@@ -0,0 +1,17 @@
+
+pragma solidity 0.6.6;
+
+import "../reserves/ISanityRate.sol";
+
+
+contract MockChainLinkSanityRate is ISanityRate {
+    uint256 public latestAnswerValue;
+
+    function setLatestKncToEthRate(uint256 _kncEthRate) external {
+        latestAnswerValue = _kncEthRate;
+    }
+
+    function latestAnswer() external view override returns (uint256) {
+        return latestAnswerValue;
+    }
+}
```

Comment:
CRLF

Reply:
fixed in https://github.com/KyberNetwork/kyber_protocol_sc/pull/11/commits/3991b8c4f2e513841e58bdcc7f9a7fc87a8c9e04